### PR TITLE
Fixes tablets losing their colours after initializing

### PR DESCRIPTION
**Pull request by Geevies**
ye

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
!bugfix

File: code/modules/modular_computers/computers/modular_computer/core.dm [hunk 1]
```diff
@@ -89,8 +89,6 @@
 		icon_state = icon_state_broken
 		add_overlay("broken")
 		return
-	else
-		icon_state = initial(icon_state)
```
Comment (Sindorman) [inline code/modules/modular_computers/computers/modular_computer/core.dm]: why did you remove this? It will make all computers have unpowered icon state all the time

Comment (Geevies) [reply code/modules/modular_computers/computers/modular_computer/core.dm]: This is the exact line of code that caused the sprite to toggle back to default. Far as I could see in my testing, it made literally no difference, as the sprite still functioned just fine. If you want, I can put the new icon_state that gets produced in a stored variable and then change the icon_state into that stored variable?

Comment (MarinaGryphon) [reply code/modules/modular_computers/computers/modular_computer/core.dm]: The unpowered icon state is the same as the powered icon state, I believe, just without the powered overlay.

Comment (Geevies) [reply code/modules/modular_computers/computers/modular_computer/core.dm]: Yeah, both sprites are the exact same. The powered on screen uses an overlay, it doesn't re-do the sprite each time, I believe.
